frontend/src/metabase/parameters/components/ParameterValuePicker/ParameterValuePicker.tsx

```diff
@@ -89,7 +90,7 @@ export function ParameterValuePicker(props: ParameterValuePickerProps) {
   // The fallback
   return (
     <DefaultParameterValueWidget
-      parameter={DEPRECATED_getAmendedParameter(tag, parameter)}
+      parameter={DEPRECATED_getAmendedParameter(tag, parameter) as UiParameter}
```

Comment:
Ran into a TS error here - for some reason `getAmendedParameter` doesn't actually return anything except a 3-key object, so I wasn't sure what to do with it besides casting for now. It seems like this component could use some love